package.json

```diff
@@ -52,6 +52,7 @@
     "promise": "6.x.x",
     "request": "2.x.x",
     "specberus": "1.x.x",
+    "mkdirp": "0.x.x",
```

Comment:
To match the other versions, this should be `0.5.x` instead.
Would that be a problem for you?